app/javascript/src/components/Dashboard/Notes/Home.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+import { Button, PageLoader } from "neetoui";
+import { Container, Header } from "neetoui/layouts";
+
+import notesApi from "apis/notes";
+import EmptyState from "components/Common/EmptyState";
+
+import NoteList from "./NoteList";
+import NewNotePane from "./Pane/Create";
+
+const Home = () => {
```

Comment:
I think calling this Notes and naming this file `index.jsx` is much more sensible.

Reply:
Yes, that seems more elegant. I will rectify this at once.